### PR TITLE
Improve `.gitattributes`

### PR DESCRIPTION
**Pull request by flobernd**
Especially improves cross-platform compatibility for Windows by defining a sane LF vs CRLF handling.

### CODE REVIEW

**Review by miguelgrinberg — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -1,3 +1,239 @@
+### Header #########################################################################################
+
+# Author: Florian Bernd
```
Comment (pquentin) [inline .gitattributes:3]: Is this needed?

**Review by pquentin — APPROVED** [review]
Thanks! Does this come from an existing template?

File: .gitattributes (hunk 1 above)
Comment (flobernd) [reply .gitattributes:3]: Ah yes, I copied it from one of my personal templates. We can of course remove the header 😋

Comment (flobernd) [inline .gitattributes:3]:
```suggestion
```

**Pull request merged**